content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -99,26 +99,29 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 
 If you are on Windows and using [Powershell Gallery](https://www.powershellgallery.com/) package manager, you can install and update kubectl with Powershell.
 
-To install:
-* Run the installation commands (making sure to specify a DownloadLocation):
+1. Run the installation commands (making sure to specify a `DownloadLocation`):
 
     ```
     Install-Script -Name install-kubectl -Scope CurrentUser -Force
     install-kubectl.ps1 [-DownloadLocation <path>]
     ```
     
-    {{< note >}}
-    **Note:** If you do not specify a DownloadLocation, kubectl will be installed in the user's temp Directory.
-    {{< /note >}}
-    The installer creates $HOME/.kube and instructs it to create a config file
-To update:
-* Run the update commands:
+{{< note >}}
+**Note:** If you do not specify a `DownloadLocation`, `kubectl` will be installed in the user's temp Directory.
+{{< /note >}}
+    
+The installer creates `$HOME/.kube` and instructs it to create a config file
+
+2. Test to ensure the version you installed is sufficiently up-to-date:
 
     ```
-    re-run Install-Script to update the installer
-    re-run install-kubectl.ps1 to install latest binaries
+    kubectl version
     ```
 
+{{< note >}}
+**Note:** Updating the installation consists in running again the two commands of the step 1.
```

Comment:
This sentence is not grammatically correct.  Perhaps:
Updating the installation is performed by rerunning the two commands listed in step 1.